src/table/utils/handle-key-press.ts

```diff
@@ -188,15 +202,23 @@ export const handleBodyKeyDown = ({
   const isSelectionMode = selectionsAPI.isModal();
   // Adjust the cellCoord depending on the totals position
   const firstBodyRowIdx = totalsPosition === 'top' ? 2 : 1;
+
   const cellCoord: [number, number] = [cell.rawRowIdx + firstBodyRowIdx, cell.rawColIdx];
   // Make sure you can't navigate to header (and totals) in selection mode
-  const topAllowedRow = isSelectionMode ? firstBodyRowIdx : 0;
+  const allowedRows = {
+    selected: {
+      top: isSelectionMode ? firstBodyRowIdx : 0,
+      bottom: isSelectionMode && totalsPosition === 'bottom' ? 1 : 0,
+    },
+    top: firstBodyRowIdx,
+    bottom: totalsPosition === 'bottom' ? 2 : 1,
+  };
```

Comment:
This should not be needed, on value for top and bottom should be enough. Is there some case that it isn't working for? I tried to change it to this
```suggestion
  const allowedRows = {
    top: isSelectionMode ? firstBodyRowIdx : 0,
    bottom: isSelectionMode && totalsPosition === 'bottom' ? 1 : 0,
  };
```
and it seemed to work for me